### PR TITLE
mc mirror -watch  event http://10.0.0.1:9000/test2/1.png sourceURLFull http://10.0.0.1:9000/test1fix eventPath not in source

### PR DESCRIPTION
**Pull request by yyimen**
mc mirror -watch minio/test1 /tmp/test1
eventPath http://10.0.0.1:9000/test2/1.png

add eventPath filter

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (klauspost) [inline .vscode/launch.json:1]: Remove please.

File: cmd/mirror-main.go [hunk 2]
```diff
@@ -513,6 +513,10 @@ func (mj *mirrorJob) watchMirrorEvents(ctx context.Context, events []EventInfo)
 		// build target path, it is the relative of the eventPath with the sourceUrl
 		// joined to the targetURL.
 		sourceSuffix := strings.TrimPrefix(eventPath, sourceURLFull)
+		//ymfix Skip the object,event http://10.0.0.1:9000/test2/1.png sourceURLFull http://10.0.0.1:9000/test1
```
Comment (klauspost) [inline cmd/mirror-main.go:516]: ?

File: cmd/watch-main.go [hunk 3]
```diff
@@ -152,6 +152,8 @@ func mainWatch(cliCtx *cli.Context) error {
 	if pErr != nil {
 		fatalIf(pErr.Trace(), "Unable to parse the provided url.")
 	}
+	//ymfix
```
Comment (klauspost) [inline cmd/watch-main.go:155]: ? Remove

File: cmd/watch-main.go [hunk 4]
```diff
@@ -189,6 +191,11 @@ func mainWatch(cliCtx *cli.Context) error {
 					return
 				}
 				for _, event := range events {
+					//ymfix
```
Comment (klauspost) [inline cmd/watch-main.go:194]: Remove

**Review by klauspost — CHANGES_REQUESTED** [review]
Please describe what you are trying to achieve and how this issue is affecting you.

**Comment by harshavardhana** [comment]
Stale PR

**Pull request closed without merging**